boards/riscv/esp32c3_luatos_core/esp32c3_luatos_core.dtsi

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2023 YuLong Yao<feilongphone@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <espressif/esp32c3/esp32c3_mini_n4.dtsi>
+#include "esp32c3_luatos_core-pinctrl.dtsi"
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "esp32c3_luatos_core";
+	compatible = "espressif,esp32c3";
+
+	aliases {
+		sw0 = &user_button1;
+		i2c-0 = &i2c0;
+		watchdog0 = &wdt0;
+		led0 = &led_d4;
+		led1 = &led_d5;
+	};
+
+	gpio_keys {
+		compatible = "gpio-keys";
+		user_button1: button_1 {
+			label = "User SW1";
+			gpios = <&gpio0 9 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			zephyr,code = <INPUT_KEY_0>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_d4: led_1 {
+			label = "D4";
+			gpios = <&gpio0 12 GPIO_ACTIVE_HIGH>;
+		};
+		led_d5: led_2 {
+			label = "D5";
+			gpios = <&gpio0 13 GPIO_ACTIVE_HIGH>;
+		};
+	};
+};
+
+&cpu0 {
+	clock-frequency = <ESP32_CLK_CPU_160M>;
+};
+
+&uart0 {
+	status = "okay";
+	current-speed = <115200>;
+	pinctrl-0 = <&uart0_default>;
+	pinctrl-names = "default";
+};
+
+&usb_serial {
+	/* requires resoldering of resistors on the board */
```

Comment:
You can remove `usb_serial` entry as it is disabled by default. In case you want to keep it, please, remove the comment as it is only for `esp32c3_devkitm` board.

Reply:
removed